packages/commerce-sdk-react/src/auth/storage.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright (c) 2022, Salesforce, Inc.
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import Cookies from 'js-cookie'
+
+export abstract class BaseStorage {
+    abstract set(key: string, value: string, options?: any): void
```

Comment:
```suggestion
    abstract set(key: string, value: string, options?: unknown): void
```
Prefer `unknown` to `any` for better type safety. 😉

Reply:
✅